code/core/template/stories/preview.ts

```diff
@@ -61,9 +62,6 @@ export const initialGlobals = {
 };
 
 export const globalTypes = {
-  foo: { defaultValue: 'fooDefaultValue' },
-  bar: { defaultValue: 'barDefaultValue' },
```

Comment:
Can you explain this change?

Reply:
`globalTypes.<x>.defaultValue` is not supported since you have done the `globals/initialGlobals` work. We only support `initialGlobals.<x>` instead as far as I remember.